Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -520,6 +520,11 @@ div.bocList div.bocListTableBlock div.bocListTableContainer table.bocListTable d
 /*endregion*/
 
 /*region Validation layout*/
+.bocListDataColumnDeclarationValidationFailureIndicator
+{
+  width: 3em;
```

Comment:
Why would we take the size of the image twice?

Since we know the exact structure of the CSS we could also use `width: calc(var(--remotion-themed-img-width) + 3 * var(--remotion-themed-spacing) + 2 * var(--remotion-themed-border-width))` which makes the td exactly the right size.

With a screenshot test we can ensure that column size does not increase when a validation failure indicator does not cause a layout shift. This would provide feedback in the future if CSS is changed that would affect this spacing.